lib/perl/Genome/Model/Command/Define.pm

```diff
@@ -24,6 +24,7 @@ sub _build_sub_command {
 
     my $model_subclass_name = $class_name;
     $model_subclass_name =~ s/::Command::Define::/::/;
+    return if eval{ $model_subclass_name->do_not_create_define_command; };
```

Comment:
I didn't want to pollute the main model namespace with this function, hence the eval. I was going to test if the model class isa 'ModelDeprecated', but that includes a bunch of models that actually aren't deprecated :facepunch: I like the try::tiny addition. I envision this being removed when we are done removing the truly deprecated model types.